site/learn/expert-advice/identity-basics/try-before-you-buy.md

```diff
@@ -0,0 +1,90 @@
+
+For example, you may be looking at a few different authentication providers and have one that you've decided is the best candidate for your application. After all, it checks all the boxes! But when your developers start reading the documentation and actually build out a small prototype, they may find the library for your application's language doesn't support [passwordless authentication](/docs/v1/tech/guides/passwordless/). If you had committed to this authentication provider before discovering this, you might have already heavily integrated it into your application—which needs passwordless authentication.
+
+This would be a huge problem leading to a major rewrite or possibly having to drop a key feature of your app. However, if you had built a prototype using a free trial and discovered this limitation ahead of time, you could have simply moved on to choosing a different provider.
+
+### Better Understanding of Costs
+
+Many API-based products such as authentication providers are usage-based, which means as your usage increases, the pricing changes. While looking at the pricing table may have you convinced that you'll be on a free or low-cost plan, actually building out a prototype might reveal otherwise.
+
+There are all sorts of potential hidden costs or costs that you may not have considered from just thinking about your application at a high level. Knowing what this product costs on average for other companies of your scale is helpful.
+
+For example, if you're working with an authentication provider that charges you on a per-authentication basis, you might take a look at your users and how active they are, and estimate how many people will be authenticating every month. With this math, you might conclude that you're well under the threshold for the authentication provider's free plan.
+
+But when you build a prototype, you might notice that every incorrect password attempt counts as authentication, demonstrating that your previous estimates might be severely different from what you should expect for actual usage. You might learn that multiple connections to enterprise identity stores, such as SAML IdPs, cost an arm and a leg.
+
+Catching surprises like this during a free trial can help you know how to factor your costs correctly, as opposed to finding out when your integration is already in production. It allows your development team a chance to think up different ways to architect your solution to fit within these constraints, or even choose a different provider if a workaround or change in architecture isn't going to be possible.
```

Comment:
Catching surprises like this during a free trial can help you ~know~ **_understand_** how